database/tables/submissions.pg

```diff
@@ -12,6 +12,7 @@ columns
     grading_method: enum_grading_method
     grading_requested_at: timestamp with time zone
     id: bigint not null default nextval('submissions_id_seq'::regclass)
+    manual_grading_user: bigint
```

Comment:
We are introducing a lot of complexity by allowing a user to have multiple submissions being graded simultaneously. Would it be cleaner to only allow a single question/submission to be graded by any given user at a time?

Reply:
The instance_question that is used in the JOIN will be locked on a FOR UPDATE.

However, on the topic of general locking logic so that items cannot be marked in parallel by multiple users, we can move that to the instance_question level. The lock was added to the submission level because the idea of marking more than one submission per instance question was discussed. The impression that I get is this is no longer the case, so it will be moved to the instance question level.